src/util/SimulationUtils.cpp

```diff
@@ -0,0 +1,269 @@
+// Copyright 2020 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "SimulationUtils.h"
+#include "crypto/SHA.h"
+#include "crypto/SignerKey.h"
+#include "transactions/TransactionUtils.h"
+
+namespace stellar
+{
+namespace SimulationUtils
+{
+SecretKey
+getNewSecret(AccountID const& key, uint32_t n)
+{
+    return SecretKey::fromSeed(
+        sha256(KeyUtils::toStrKey(key) + std::to_string(n)));
+}
+
+int64_t
+getNewOfferID(int64_t offerId, uint32_t n)
+{
+    if (n == 0 || offerId == 0)
```

Comment:
The code used to throw on offerID == 0, but I changed it since 0 is a valid offer ID when creating/deleting offers used in operation simulation code (see `SimulationUtils::updateOperation`)